asciidoc/workloads/azure/backend/java/quickstart_java.adoc

```diff
@@ -0,0 +1,196 @@
+= Getting started with the Java Rest API application
+:description: Getting started with the Java Rest API application
+:keywords: java, rest, api, application, spring boot, authorization, docker, swagger, auth0, settings, get started
+
+== Using the repository to build the Java Spring Boot Rest API application
+
+=== Running the application locally
+
+1. Clone one of the Java workloads to your local machine from one of the following repos
```

Comment:
This should be a heading. 

I think the structure of the current md doc has caused formatting issues in asciidoc. It doesn't seem to make sense that the individual headings follow a numbered list.